epf/Epf.cpp

```diff
@@ -226,10 +239,21 @@ PointCount Epf::createFileInfo(const StringList& input, StringList dimNames,
         s->setOptions(opts);
 
         QuickInfo qi = s->preview();
-
         if (!qi.valid())
             throw "Couldn't get quick info for '" + filename + "'.";
 
+        // Get scale values from the reader if they exist.
+        pdal::MetadataNode root = s->getMetadata();
+        pdal::MetadataNode m = root.findChild("scale_x");
+        if (m.valid())
```

Comment:
Shouldn't this look like:
```
if (m_b.scale[0] == -1) m_b.scale[0] = m.value<double>();
else m_b.scale[0] = std::min(m_b.scale[0], m.value<double>());
```

Wouldn't we override a more precise scale like `0.001` with later occurring `0.01` by using `max()` as written, losing precision in the output?

Reply:
Yes. That's the idea.

Reply:
So how do I index a dataset containing scales of both `0.001` and `0.01` losslessly?  Does it need to be set manually - I'm not seeing that option.  This seems backwards to me.

Reply:
This current use case is visualization through QGIS, not lossless storage. You have a choice, modify the scaling or fail.

Reply:
That's kind of my point though - clipping a scale of 0.001 to 0.01 can result in very noticeable side-effects for visualization.  And regarding "modify the scaling", that's why I was asking about a manual option, how do I do this?  I don't see an option in untwine.cpp.